tests/PHPStan/Analyser/data/constant-array-type-identical.php

```diff
@@ -21,8 +21,8 @@ public function doFoo(string $s): void
 		assertType('false', [1] === [2]);
 		assertType('false', [1] !== [1]);
 		assertType('true', [1] !== [2]);
-		assertType('bool', [$s] === [$s]);
-		assertType('bool', [$s] !== [$s]);
+		assertType('true', [$s] === [$s]);
```

Comment:
Please add a test for two different strings. Add `string $t` to the parameters and then test `[$s] === [$t]` and `[$s] !== [$t]`.

Reply:
Turns out this change was wrong, so I reverted it.